docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -177,3 +177,78 @@ var token = await client.authenticationKeyManager?.get();
 ```
 
 The above example retrieves the token from the key manager and stores it in the `token` variable.
+
+## Authentication schemes
+
+By default Serverpod will pass the authentication token from client to server in accordance with the HTTP `authorization` header standard with the `basic` scheme name and encoding. This is securely transferred as the connection is TLS encrypted.
+
+The default implementation encodes and wraps the user-provided token in a `basic` scheme which is automatically unwrapped on the server side before being handed to the user-provided authentication handler described above.
+
+In other words the default transport implementation is "invisible" to user code.
+
+### Implementing your own authentication scheme
+
+If you are implementing your own authentication and are using the `basic` scheme, note that this is supported but will be automatically unwrapped i.e. decoded on the server side before behing handed to your `AuthenticationHandler` implementation. It will in this case receive the decoded auth key value after the `basic` scheme name.
```

Comment:
Sugestion: change `...server side before behing handed...` to `...server side before being handed...`